packages/api/src/clickhouse/index.ts

```diff
@@ -955,7 +955,14 @@ export const buildMetricSeriesQuery = async ({
           })(${isRate ? 'rate' : 'value'}) as data`,
     );
   } else {
-    logger.error(`Unsupported data type: ${dataType}`);
+    if (dataType === MetricsDataType.Histogram) {
+      if (!['p50', 'p90', 'p95', 'p99'].includes(aggFn)) {
+        throw new Error(`Unsupported aggFn for Histogram: ${aggFn}`);
+      }
+      selectClause.push(`max(value) as data`);
+    } else {
+      logger.error(`Unsupported data type (994): ${dataType} `);
+    }
```

Comment:
can clean this up:
```
else if (dataType === MetricsDataType.Histogram) {
  if (!['p50', 'p90', 'p95', 'p99'].includes(aggFn)) {
    throw new Error(`Unsupported aggFn for Histogram: ${aggFn}`);
  }
} else {
  // I think we should probably throw instead of bypassing it in this case
  throw new Error(`Unsupported data type: ${dataType}`);
}
```